Client/Frontend/UserContent/UserScripts/PlaylistFolderSharing.js

```diff
@@ -0,0 +1,25 @@
+window.__firefox__.includeOnce("PlaylistFolderSharing", function() {
+  function sendMessage(pageUrl) {
+      if (window.webkit.messageHandlers.$<handler>) {
+          window.webkit.messageHandlers.$<handler>.postMessage({
+            "securitytoken": "$<security_token>",
```

Comment:
I don't think we compare this token anywhere. Maybe not a big deal, since the handler name is randomized.

Reply:
Fixed: https://github.com/brave/brave-ios/pull/5798/commits/4a9058fc1dd678ea49d615caedd7fdc9c4c6e37b